<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtRPT examples</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../QtRPT/imagesRpt.qrc">
    <normaloff>:/QtRptDesigner/images/QtRptDesigner.ico</normaloff>:/QtRptDesigner/images/QtRptDesigner.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Select example</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="rBtn1">
           <property name="text">
            <string>Invoice report (data from grid)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn2">
           <property name="text">
            <string>Two datasource report</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn3">
           <property name="text">
            <string>Embedded report widget</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn4">
           <property name="text">
            <string>Report with condition (Printable and highlighting fields)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn5">
           <property name="text">
            <string>Report with mathematics and aggregate functions</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn6a">
           <property name="text">
            <string>Report with grouping data in Master Data Band 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn6b">
           <property name="text">
            <string>Report with grouping data in Master Data Band 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn7">
           <property name="text">
            <string>Report with Diagram</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn8">
           <property name="text">
            <string>Report with AutoHeight</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn9">
           <property name="text">
            <string>Report with Drawing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn10">
           <property name="text">
            <string>Report with SQL query to SQLITE DB</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn11">
           <property name="text">
            <string>Barcode generator</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn12">
           <property name="text">
            <string>Report with RichText Fields</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtnRussian">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string>Russian invaders</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn13">
           <property name="text">
            <string>Report's control from user application</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn14">
           <property name="text">
            <string>Creation of the report from the user application without XML file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn15">
           <property name="text">
            <string>Generate PDF file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn16">
           <property name="text">
            <string>Application defined SQL parameters and images from SQLITE DB table</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn17">
           <property name="text">
            <string>Buil in functions and system variables</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rBtn18">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>CrossTab report</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="resurce.qrc">:/images/Logo128.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnShow">
           <property name="text">
            <string>Show</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>578</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPrint">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../QtRPT/imagesRpt.qrc"/>
  <include location="../QtRPT/imagesRpt.qrc"/>
  <include location="../QtRPT/imagesRpt.qrc"/>
  <include location="resurce.qrc"/>
 </resources>
 <connections/>
</ui>
